bark/runtime/viewer/buffered_viewer.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2021 fortiss GmbH
+#
+# Authors: Julian Bernhard, Klemens Esterle, Patrick Hart and
+# Tobias Kessler
+#
+# This work is licensed under the terms of the MIT license.
+# For a copy, see <https://opensource.org/licenses/MIT>.
+import math
+import numpy as np
+from bark.core.geometry import *
+from bark.core.models.dynamic import *
+from bark.core.world.opendrive import *
+from bark.core.world.goal_definition import *
+from bark.core.world.renderer import *
+from bark.runtime.commons.parameters import ParameterServer
+
+
+class BufferedViewer:
```

Comment:
what's the difference to the Viewer Class? will we maintain both? or should the viewers inherit from this?

Reply:
This BufferedViewer just writes (geometric) primitives into the renderer's buffer. The buffer can then, e.g., be visualized using BARKSCAPE.